server/DBinstallation.md

```diff
@@ -0,0 +1,3 @@
+npm install sequelize sequelize-cli pg pg-hstore
+
+sequelize init
```

Comment:
These dependencies are on package.json, so there shouldn't be necessary to install them again
Add an "db:initialize" script to package.json so that the user can do that by running `npm run sequelize init`

Reply:
I have updated the DBInstallation.md file, adding instructions on how to setup the database properly. I called the script "setupDB" though. I don't know if that matters or not.

("npm run setupDB")